libraries/lib-music-information-retrieval/MirDsp.cpp

```diff
@@ -84,14 +84,16 @@ std::vector<float> GetMovingAverage(const std::vector<float>& x, double hopRate)
 }
 } // namespace
 
-std::vector<float> GetNormalizedCircularAutocorr(std::vector<float> x)
+std::vector<float>
+GetNormalizedCircularAutocorr(std::vector<float> ux /* unaligned x*/)
```

Comment:
I think, but I might be wrong, that const ref should be used now in this place. `x` is no longer used in place, and the ownership of `x` is not transferred on the main branch